themes/root/templates/RecordDriver/AbstractBase/export-refworks.phtml

```diff
@@ -40,19 +40,15 @@ if (!empty($journalTitle)) {
     }
 }
 
-$author = $this->driver->tryMethod('getPrimaryAuthor');
-if (!empty($author)) {
-    echo "A1 $author\n";
+foreach ($this->driver->tryMethod('getPrimaryAuthors', [], []) as $current) {
+    echo "A1 $current\n";
 }
 
-$secondaryAuthors = $this->driver->tryMethod('getSecondaryAuthors');
-if (is_array($secondaryAuthors)) {
-    foreach ($secondaryAuthors as $current) {
-        echo "A2 $current\n";
-    }
+foreach ($this->driver->tryMethod('getSecondaryAuthors', [], []) as $current) {
+    echo "A2 $current\n";
 }
 
-$languages = $this->driver->tryMethod('getLanguages');
+$languages = $this->driver->tryMethod('getLanguages', [], []);
```

Comment:
Looks like you started to simplify this but didn't remove the is_array check below...

Reply:
Oops. I started it but meant to revert to avoid scope creep.